OTVision/helpers/files.py

```diff
@@ -146,13 +151,20 @@ def _remove_dir(dir_to_remove: Path) -> None:
     dir_to_remove.rmdir()
 
 
-def read_json(json_file: Path, filetype: str = ".json") -> dict:
+def read_json(
+    json_file: Path,
+    filetype: str = ".json",
+    decompress: bool = True,
```

Comment:
Should we add this to the config?

Reply:
As it is a helper method, I would not add it to the config. After we defined the output format. We can decide which output formats should be compressed and add one config option for each file format.